xml/Microsoft.Extensions.Configuration/IConfiguration.xml

```diff
@@ -98,8 +98,8 @@ For more information, see [Configuration in .NET](/dotnet/core/extensions/config
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>Returns a <see cref="T:Microsoft.Extensions.Primitives.IChangeToken" /> that can be used to observe when this configuration is reloaded.</summary>
-        <returns>A <see cref="T:Microsoft.Extensions.Primitives.IChangeToken" />.</returns>
+        <summary>Attempts to get an <see cref="T:Microsoft.Extensions.Primitives.IChangeToken" /> for change tracking.</summary>
+        <returns>An <see cref="T:Microsoft.Extensions.Primitives.IChangeToken" /> token if this provider supports change tracking, <see langword="null"/> otherwise.</returns>
         <remarks>To be added.</remarks>
```

Comment:
@tacosontitan 
Could you please fill the remarks element with something like `GetReloadToken is a method that can be used to get a change token that notifies when the configuration is reloaded. A change token is a way to track changes in an object, such as a file, a cache entry, or an option. `?

Reply:
I've updated the PR to include remarks as requested.